charts/stable/omada-controller/values.yaml

```diff
@@ -26,15 +26,48 @@ service:
     ports:
       http:
         port: 8043
+  management:
```

Comment:
I don't think it really adds much value to run three separate Services for this. Could you please take a look at how the Unifi Controller chart (https://github.com/k8s-at-home/charts/blob/master/charts/stable/unifi/values.yaml#L47) handles this, and implement it in a similar way?